selinux/README.md

```diff
@@ -1,139 +1,157 @@
-# CSI WekaFS SELinux Support
+# Add SELinux support
 
-## General Information
-When installing Weka CSI plugin on SELinux-enabled Kubernetes cluster, pods might be denied access
-to the persistent volumes provisioned on top of Weka filesystem.
+## Overview
+Security-Enhanced Linux (SELinux) is a Linux kernel security module designed to enforce access control security policies, including Mandatory Access Controls (MAC).
 
-The reason behind this is a lack of permissions for containers to access objects stored on Weka cluster.
+This section applies exclusively to SELinux-enabled deployments that are not based on the Red Hat OpenShift Container Platform (OCP). In OCP environments, all required configurations are automatically handled.
 
-In this directory you can find a custom policy that provides all the necessary security configuration to optionally 
-enable pod access to WekaFS-based Persistent Volumes, and it should be applied 
-on each Kubernetes worker node that is intended to service WekaFS-based persistent volumes.
+The purpose of this section is to address permission issues that prevent containers from accessing objects stored on the WEKA cluster.
 
-The provided policy allows processes with `container_t` seclabel to access objects having `wekafs_t` label (which is set for all files and directories of mounted CSI volumes).
+To add SELinux support, perform the following procedures:
+
```

Comment:
Add a note that says. 
On OpenShift Container Platform (OCP) with CoreOS-based workers, the selinux policy is automatically installed in form of a `machineconfig`. 
For other types of SELinux-enabled workers (including OCP RedHat workers), installation must be performed manually.

For CoreOS-based workers on OCP, skip step 1